handlers.go

```diff
@@ -269,6 +276,15 @@ func (s *Server) ServeAPI() error {
 	admin.POST("/cm/transfer/restart/:chanid", s.handleTransferRestart)
 	admin.POST("/cm/repinall/:shuttle", s.handleShuttleRepinAll)
 
+	//	peering
+	adminPeering := admin.Group("/peering")
+	adminPeering.POST("/peers/add", s.handlePeeringPeersAdd)
+	adminPeering.DELETE("/peers/remove", s.handlePeeringPeersRemove)
+	adminPeering.GET("/peers/list", s.handlePeeringPeersList)
```

Comment:
@alvin-reyes not sure there is a need for the `peer` in the route, same with adding `add`, `remove` and `list` in the route 
POST /peering - is enough for add
GET /Perring - is enough for list
DELETE /peering -  enough for delete

Reply:
Given how `peering/peers` are structured in ipfs, IMO it make sense to have `/peers` because it directly calls the object we are operating in e.i `peers`/

I modified the implementation to the following instead: 

- POST /peering/peers - for adding a list of peers
- DELETE /peering/peers - to remove a peer
- GET /peering/peers - list of peers

And these are the `Peering` level endpoints.

- POST /peering/stop 
- POST /peering/start
- GET /peering/status